remote_engine_client/src/lib.rs

```diff
@@ -93,12 +94,14 @@ pub mod error {
         },
 
         #[snafu(display(
-            "Failed to query from table in server, table_idents:{:?}, code:{}, msg:{}",
+            "Failed to query from table in server, table_idents:{:?}, endpoint:{}, code:{}, msg:{}",
             table_idents,
+            endpoint.to_string(),
```

Comment:
Isn't `Debug` trait derived by `Endpoint`?

Reply:
```
impl ToString for Endpoint {
    fn to_string(&self) -> String {
        format!("{}:{}", self.addr, self.port)
    }
}

```
This is more readable then debug format.